mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -14,23 +14,38 @@
 import xarray as xr
 import datetime
 from netCDF4 import Dataset as netcdf_dataset
+import os.path
 
 from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
 from ..shared.plot.plotting import plot_global_comparison
 from ..shared.interpolation.interpolate import interp_fields, init_tree
 from ..shared.constants import constants
 
+from ..shared.io import StreamsFile
 
 def ocn_modelvsobs(config, field):
 
     """
     Plots a comparison of ACME/MPAS output to SST or MLD observations
 
-    Authors: Luke Van Roekel and Milena Veneziani
-    Modified: 10/24/2016
+    Authors: Luke Van Roekel, Milena Veneziani, Xylar Asay-Davis
+    Modified: 10/27/2016
     """
 
+    # read parameters from config file
     indir = config.get('paths', 'archive_dir_ocn')
+
+    streams_filename = config.get('input', 'ocean_streams_filename')
+    streams = StreamsFile(streams_filename, streamsdir=indir)
+
+    # get a list of timeSeriesStats output files from the streams file,
+    # reading only those that are between the start and end dates
+    startDate = config.get('time', 'climo_start_date')
+    endDate = config.get('time', 'climo_end_date')
+    infiles = streams.readpath('timeSeriesStatsOutput',
+                               startDate=startDate, endDate=endDate)
```

Comment:
Why not inline this so that

```python
infiles = streams.readpath('timeSeriesStatsOutput',
                                          startDate=config.get('time', 'climo_start_date'), 
                                          endDate=config.get('time', 'climo_end_date'))
```

This may be cleaner, but that is arguable.  Note, I couldn't align the example above in markdown.